### ISSUE TITLE
SpaceInHtmlTag customisation

### ISSUE THREAD

**Issue by TomasBarry** [Shopify/erb-lint#125]
When the `SpaceInHtmlTag` rule is enabled, the following code is flagged:

```erb
<img
  alt='...'
  className='...'
/>
```

I realise that I could just disable this rule but then the following code is accepted:

```erb
<img     alt='...'              className='...' />
```

Which is undesirable.

Is it possible to customise the `SpaceInHtmlTag` rule such that scenario 1 is accepted but scenario 2 is not? Or do we have to make a choice on what we want to accept?

**Comment by glenpike** [comment]
I would also like to customise the exclusions from this as an empty alt tag: `alt=""` is recommended for accessibility compliance with some image types - https://www.w3.org/WAI/tutorials/images/decorative/

**Comment by coder2000** [comment]
I would like to see additional characters supported. Some js libraries like alpine use special characters like `@` and it causes issues.

**Comment by CodeF53** [comment]
I also had to disable this rule due to use of Alpine's `@` event syntax.
